### DISCUSSION TITLE
cdc能否指定某个时间点开始增量拉取

### DISCUSSION THREAD

**Discussion by stgztsw** [apache/flink-cdc#1562] [Q&A]
scan.startup.mode只有initial和 latest-offset两种模式，但是大多数场景，我们需要从某个时间点开始消费，cdc有没有类似kafka这种指定从某个时间点开始拉取数据的功能啊

**Comment by gtk96** [comment]
Follow this pr
https://github.com/ververica/flink-cdc-connectors/pull/1157
